docs/source/membership/membership.md

```diff
@@ -132,94 +314,52 @@ operation, all organizations on the channel must agree to it. Therefore, the
- --- as well as which of these members are authorized to perform
- administrative tasks (e.g., creating a channel).
-
- * **Channel MSP:** It is important for a channel to maintain the MSPs of its members
- separately. A channel provides private communications between a particular set of
- organizations which in turn have administrative control over it. Channel policies
- interpreted in the context of that channel's MSPs define who has ability to
- participate in certain action on the channel, e.g., adding organizations, or
- instantiating chaincodes. Note that there is no necessary relationship between
- the permission to administrate a channel and the ability to administrate the
- network configuration channel (or any other channel). Administrative rights
- exist within the scope of what is being administrated (unless the rules have
- been written otherwise --- see the discussion of the `ROLE` attribute below).
-
- * **Peer MSP:** This local MSP is defined on the file system of each peer and there is a
- single MSP instance for each peer. Conceptually, it performs exactly the same function
- as channel MSPs with the restriction that it only applies to the peer where it is defined.
- An example of an action whose authorization is evaluated using the peer's local MSP is
- the installation of a chaincode on the peer.
-
- * **Orderer MSP:** Like a peer MSP, an orderer local MSP is also defined on the file system
- of the node and only applies to that node. Like peer nodes, orderers are also owned by a single
- organization and therefore have a single MSP to list the actors or nodes it trusts.
-
-### MSP Structure
+## MSP Structure
 
 So far, you've seen that the most important element of an MSP are the specification
 of the root or intermediate CAs that are used to establish an actor's or node's
 membership in the respective organization. There are, however, more elements that are
 used in conjunction with these two to assist with membership functions.
 
-![MSP4](./membership.diagram.5.png)
+A local MSP folder contains the following sub-folders:
 
-*The figure above shows how a local MSP is stored on a local filesystem. Even though
-channel MSPs are not physically structured in exactly this way, it's still a helpful
-way to think about them.*
+![MSP6](./membership.diagram.6.png)
 
-As you can see, there are nine elements to an MSP. It's easiest to think of these elements
-in a directory structure, where the MSP name is the root folder name with each
-subfolder representing different elements of an MSP configuration.
+*The figure above shows the subfolders in a local MSP on the file system*
 
-Let's describe these folders in a little more detail and see why they are important.
+* **config.yaml:**  Used to configure the identity classification feature
+  in Fabric by enabling "Node OUs" and defining the accepted roles.
+
+* **admincerts (Deprecated from Fabric v1.4.3 and higher):** This folder
```

Comment:
This information belongs with the rest of the implementation details, imo. And personally I might put it first in a section about implementation.

Reply:
I moved it up to just after What is an MSP?

Reply:
But ended up moving it back down because it depends on understanding local msps, channel msps, organizations and OU. :-(